src/main/java/com/amazonaws/encryptionsdk/AwsCrypto.java

```diff
@@ -281,7 +281,11 @@ public long estimateCiphertextSize(
     /**
      * Calls {@link #encryptData(MasterKeyProvider, byte[], Map)} on the UTF-8 encoded bytes of
      * {@code plaintext} and base64 encodes the result.
+     * @deprecated Use the {@link #encryptData(MasterKeyProvider, byte[], Map)} and
+     * {@link #decryptData(MasterKeyProvider, byte[])} APIs instead. {@code encryptString} is not out-of-the-box
+     * compatible with the other Encryption SDKs and intentionally not supported by the other implementations.
      */
```

Comment:
{@code encryptString} is incompatible with the other AWS Encryption SDK language implementations. By design, this feature is not supported by the other language implementations.

Reply:
Thanks, fixing.